client/components/App.js

```diff
@@ -1,49 +1,54 @@
 //libraries
-import React, { Component } from 'react';
-import { Route } from 'react-router-dom';
-import { connect } from 'react-redux';
+import React, {Component} from 'react';
+import {Route} from 'react-router-dom';
+import {connect} from 'react-redux';
 import axios from 'axios';
 import ReactDependentScript from 'react-dependent-script';
 //import {GOOGLE_API_KEY} from '../../config';
 
 //components
-import HomeSphere from './Home-Sphere';
 import Home from './Home';
+import HomeCheck from './Home-Check';
 import Nav from './Nav';
 import Map from './Map';
 
 //redux
-import { fetchNearby } from '../redux/store';
+import {fetchNearby} from '../redux/store';
 
 //styles
 import '../styles/App.css';
 
 class App extends Component {
-  componentDidMount() {
+  constructor(props) {
+    super(props);
+    this.state = {key: 'holder'};
+  }
+
+  async componentDidMount() {
     var socket = io('http://localhost:3000');
     socket.on('updateNearby', entry => {
       console.log(entry);
     });
 
     navigator.geolocation.getCurrentPosition(position => {
-      const { latitude, longitude } = position.coords;
+      const {latitude, longitude} = position.coords;
       this.props.fetchNearby({
-        coordinate: { latitude, longitude },
+        coordinate: {latitude, longitude},
         distance: 500,
       }); //distance is in feet 5280ft = 1mi
     });
 
-    await axios
-      .get('/googlemaps')
-      .then(response => this.setState({ key: response.data }));
+    // await axios
```

Comment:
This should fix the map render:
```suggestion
       await axios
       .get('/googlemaps')
       .then(response => this.setState({key: response.data}));
```